### ISSUE TITLE
WebAssembly trapped: Exited with i32 exit status 0

### ISSUE THREAD

**Issue by AndrewBarba** [fastly/Viceroy#127]
When running a basic Swift application Viceroy 0.2.10 produces a misleading trap after a successful request:

<img width="836" alt="Screen Shot 2022-02-10 at 2 49 43 PM" src="https://user-images.githubusercontent.com/1316152/153485748-246f8280-ad75-4983-b848-925f90287241.png">

It says the error code is 0, which to me means a non-error. So I'm not sure if the Swift runtime is doing something incorrectly after it sends the response? Any extra details on the Viceroy side would be helpful and I can handle Swift side.

Application:

```swift
import Compute

@main
struct App {
    static func main() async throws {
        try await onIncomingRequest(handleIncomingRequest)
    }

    static func handleIncomingRequest(req: IncomingRequest, res: OutgoingResponse) async throws {
        try await res.status(200).send("Hello, Swift")
    }
}
```

**Comment by cratelyn** [comment]
thank you for filing this issue @AndrewBarba! you are right that a return code of 0 shouldn't be logged as an error. i've since opened and landed #128, which should address this problem.

the current `main` should no longer emit this warning. i'll wait to close this until a new `viceroy` package has been released :package: :butterfly:

**Comment by AndrewBarba** [comment]
Awesome thanks so much for the quick fix

**Comment by cratelyn** [comment]
this was released in [`0.2.11`](https://crates.io/crates/viceroy/0.2.11), which should be automatically picked up by the CLI :sparkles: i'm going to close this now, thanks again for filing this Andrew!
